### ISSUE TITLE
Long inline code causes it to overflow and render outside of the preview box

### ISSUE THREAD

**Issue by sleithdylan** [codu-code/codu#155]
## Context

- Using a 1920x1080 Monitor

## Expected Behavior

When I input a long inline code I expect it to fit within the preview container/box.

## Current Behavior

When I input a long inline code I currently see it clipping outside the preview container/box.

## Screenshots

![2TguO1X4LO](https://user-images.githubusercontent.com/30472715/217099061-40e6aa0b-99f2-4cef-813d-f239ccfdadd4.png)

## Steps to reproduce

1. Go to [https://www.codu.co/create](https://www.codu.co/create)

2. Type in a long inline code, example below:
```markdown
`sh -c "$(curl -fsSL [https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh](https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh))"`
```
3. Click **View preview** and you will see the inline code overflowing in the preview container/box

## Additional info

n/a

**Comment by NiallJoeMaher** [comment]
Thanks for adding this! ❤️  I'll add it to my queue next week if nobody takes it before then.

**Comment by sleithdylan** [comment]
@NiallJoeMaher I knew what was happening so I went ahead and created a PR for it. :)

**Comment by NiallJoeMaher** [comment]
Closed with #162, thanks @sleithdylan! ❤️
